src/metabase/upload.clj

```diff
@@ -304,15 +304,35 @@
 ;;;; |  Parsing values  |
 ;;;; +------------------+
 
+(import 'java.time.LocalDateTime)
+(import 'java.time.temporal.ChronoUnit)
```

Comment:
Oops, didn't expect my IDE to take a shortcut like this. I'll fix this up after the reviews are in.